lib/enrichers/schema.relationships.enricher.js

```diff
@@ -95,19 +104,46 @@ function enrichSchemaWithRelationships(
 function _resolveModelProperties(
   modelClass,
   isIncludeRelationships,
-  isIncludeParentRelationships
+  isIncludeParentRelationships,
+  modelPaths
 ) {
-  const model = _.isString(modelClass) ? require(modelClass) : modelClass;
+  let model;
+
+  if ( _.isString(modelClass)) {
+    if (modelPaths && modelPaths.length) {
+      for (const modelPath of modelPaths) {
+        let fullPath = path.join(process.cwd(), modelPath, modelClass);
+
+        if (!fullPath.endsWith('.js'))
```

Comment:
Wouldn't that break support of .ts when executed with ts-node? Can you skip this logic if full path ends with .ts?

Reply:
yep, good catch!

Reply:
I've added the missing check.